<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>34</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eMailLineEdit">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>30</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>63</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdLineEdit">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>92</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="conPWlineEdit">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveLogin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>75</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>141</y>
     <width>87</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Generated Key:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yourIdLineEdit">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>160</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>186</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>eMailLineEdit</tabstop>
  <tabstop>passwdLineEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>yourIdLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>login</receiver>
   <slot>loginButton_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>login</receiver>
   <slot>startButton_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>passwdLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>eMailLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>yourIdLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>login</receiver>
   <slot>cancelButton_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>login</receiver>
   <slot>saveLogin_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>conPWlineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginButton_click()</slot>
  <slot>startButton_click()</slot>
  <slot>cancelButton_click()</slot>
  <slot>saveLogin_click()</slot>
 </slots>
</ui>
